boa/src/syntax/parser/statement/declaration/hoistable_declaration.rs

```diff
@@ -0,0 +1,118 @@
+//! Hoistable declaration parsing.
```

Comment:
Maybe we should call this just `hoistable` module. It already lives in `declaration` module.